sprout/sprout.sh

```diff
@@ -30,7 +30,7 @@ LOGFILE="./sprout-manager.log"
 UPDATE_LOG="./update.log"
 GUNICORN_CMD="gunicorn --bind 127.0.0.1:${DJANGO_PORT:-8000} -w ${GUNICORN_WORKERS:-4} --access-logfile access.log --error-logfile error.log sprout.wsgi:application"
 MEMCACHED_CMD="memcached -l 127.0.0.1 -p ${MEMCACHED_PORT:-23156}"
-WORKER_CMD="./celery_runner worker --app=sprout.celery:app --concurrency=${CELERY_MAX_WORKERS:-8} --loglevel=INFO -Ofair"
+WORKER_CMD="./celery_runner worker --app=sprout.celery:app --concurrency=${CELERY_MAX_WORKERS:-2} --loglevel=INFO -Ofair"
```

Comment:
I dropped this to two since this script is used when running sprout outside of production. @izapolsk if you would rather keep this at 8 I'll reset it.